da/test/da_test.go

```diff
@@ -34,7 +33,7 @@ var (
 	testNamespaceID = types.NamespaceID{0, 1, 2, 3, 4, 5, 6, 7}
 
 	testConfig = celestia.Config{
-		BaseURL:  "http://localhost:26658",
+		BaseURL:  "http://localhost:26658", // placeholder; actually set in startMockCelestiaNodeServer
```

Comment:
If this is actually set in `startMockCelestiaNodeServer`, can this just be removed here? Does the placeholder serve any purpose?

Reply:
OK, removed.